cms/djangoapps/contentstore/views/item.py

```diff
@@ -657,6 +667,7 @@ def _delete_item(usage_key, user):
             store.update_item(course, user.id)
 
         store.delete_item(usage_key, user.id)
+        SignalHandler.item_deleted.send(sender=None, usage_key=usage_key, user_id=user.id)
```

Comment:
Does store.delete_item not emit a signal (or should it)? I'm not familiar with this SignalHandler, but it seems odd that it would be necessary to call both store.delete_item and the item_deleted signal handler.